src/Game/GlobalEvents.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "ScreenGame.h"
+#include "ScreenMainMenu.h"
+#include "WorldGeneration.h"
+#include "../Keyboard.h"
+#include "../Profiler.h"
+#include "../Screen.h"
+#include <SFML/Graphics/RenderWindow.hpp>
+#include <imgui_sfml/imgui-SFML.h>
+#include <imgui_sfml/imgui.h>
+
```

Comment:
This file shouldn't be in the Game/ folder as it is not strictly related to the game

Reply:
Doesn't really need to outside of main anyways. I could probably make it look "neater" using if statements rather than the switch case

Reply:
yeah that was from an old branch of mine

Reply:
wait ill make a new pr that makes more sense